collectors/metrics/pkg/metricsclient/metricsclient_test.go

```diff
@@ -22,7 +22,7 @@ func TestDefaultTransport(t *testing.T) {
 		TLSHandshakeTimeout: 10 * time.Second,
 		DisableKeepAlives:   true,
 	}
-	http := DefaultTransport(logger, true)
+	http := DefaultTransport(logger)
 	if http.Dial == nil || reflect.TypeOf(http) != reflect.TypeOf(want) {
```

Comment:
`http.Dial` is deprecated, `http.DialContext` instead

Reply:
Ack, will update, this was old code